tests/test_preprocessing.py

```diff
@@ -0,0 +1,605 @@
+        },
+    )
+
+    ds_out_expected = xr.Dataset(
+        {
+            "gdp": (["ssp", "region", "model", "year"], np.ones((1, 2, 2, 3))),
+            "gdppc": (["ssp", "region", "model", "year"], np.ones((1, 2, 2, 3))),
+            "pop": (["ssp", "region", "model", "year"], np.ones((1, 2, 2, 3))),
+        },
+        coords={
+            "ssp": (["ssp"], ["SSP3"]),
+            "region": (["region"], ["USA.test_region", "XWX.test_region"]),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "year": (["year"], [2021, 2022, 2023]),
+        },
+    )
+    ds_in.to_zarr(infile)
+
+    subset_USA_ssp_econ(infile, outfile)
+    ds_out_actual = xr.open_zarr(outfile)
+
+    xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+
+@pytest.mark.parametrize("recipe", ["adding_up", "risk_aversion"])
+def test_subset_USA_reduced_damages(tmp_path, recipe):
+    """
+    Test that subset_USA_reduced_damages returns a Zarr file with only regions containing USA
+    """
+    eta = 10
+    sector = "dummy_sector"
+    reduction = "cc"
+
+    d = tmp_path / "USA_econ"
+    d.mkdir()
+    if recipe == "adding_up":
+        infile = d / f"{sector}/{recipe}_{reduction}.zarr"
+        outfile = d / f"{sector}_USA/{recipe}_{reduction}.zarr"
+    else:
+        infile = d / f"{sector}/{recipe}_{reduction}_eta{eta}.zarr"
+        outfile = d / f"{sector}_USA/{recipe}_{reduction}_eta{eta}.zarr"
+
+    ds_in = xr.Dataset(
+        {
+            reduction: (
+                ["ssp", "region", "model", "year", "gcm", "rcp"],
+                np.ones((2, 3, 2, 3, 2, 2)),
+            ),
+        },
+        coords={
+            "ssp": (["ssp"], ["SSP3", "SSP4"]),
+            "region": (
+                ["region"],
+                ["ZWE.test_region", "USA.test_region", "XWX.test_region"],
+            ),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "year": (["year"], [2021, 2022, 2023]),
+            "gcm": (["gcm"], ["Jonahs_GCM", "surrogate_GCM"]),
+            "rcp": (["rcp"], ["rcp45", "rcp85"]),
+        },
+    )
+
+    ds_out_expected = xr.Dataset(
+        {
+            reduction: (
+                ["ssp", "region", "model", "year", "gcm", "rcp"],
+                np.ones((2, 2, 2, 3, 2, 2)),
+            ),
+        },
+        coords={
+            "ssp": (["ssp"], ["SSP3", "SSP4"]),
+            "region": (["region"], ["USA.test_region", "XWX.test_region"]),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "year": (["year"], [2021, 2022, 2023]),
+            "gcm": (["gcm"], ["Jonahs_GCM", "surrogate_GCM"]),
+            "rcp": (["rcp"], ["rcp45", "rcp85"]),
+        },
+    )
+
+    ds_in.to_zarr(infile, consolidated=True, mode="w")
+
+    subset_USA_reduced_damages(sector, reduction, recipe, eta, d)
+    ds_out_actual = xr.open_zarr(outfile)
+
+    xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+
+def test_sum_AMEL(tmp_path):
```

Comment:
Add a docstr here giving one line saying the one behavior this test is trying to check. At this stage, testing legacy code, I think it's okay if it's something “Tests the basic functionality of <insert function name here>“. But try to have each test try to check one behavior or have one reason to fail.

There are a couple other missing docstrs below. Ill try to call them all out.